### PR TITLE
Lego 3487 forkortelse stotte i elvis og popover

### PR DESCRIPTION
**Pull request by TrygveUrdahl**
# PR Checklist
https://elvia.atlassian.net/wiki/spaces/TEAMATOM/pages/10427498683/Review+prosess

- [ ] Bumpet package?
- [ ] Updated changelog?
- [ ] Correct date in changelog?

## Describe PR briefly:
https://elvia.atlassian.net/browse/LEGO-3487

### CODE REVIEW

File: packages/components/components/elvis-popover/src/react/elvia-popover.tsx [hunk 1]
```diff
@@ -93,6 +83,12 @@ export const Popover: FC<PopoverProps> = function ({
     updatePreferredPosition();
   }, [isShowingConnectedOverlayState, content, horizontalPosition, verticalPosition, heading]);
 
+  /* Add class to text-abbreviation element trigger element when the popover is open */
+  useEffect(() => {
+    const textAbbreviationElement = triggerRef.current?.querySelector('.e-text-abbreviation');
+    textAbbreviationElement?.classList.toggle('e-text-abbreviation---active', isShowingConnectedOverlayState);
+  }, [isShowingConnectedOverlayState]);
```
Comment (TrygveUrdahl) [inline packages/components/components/elvis-popover/src/react/elvia-popover.tsx:90]: This is not the nicest, but I prefer adding this little thing here over having to make our users make the popover controlled simply to add this one class when active 🤔

Comment (TrygveUrdahl) [reply packages/components/components/elvis-popover/src/react/elvia-popover.tsx:90]: Though it is something that won't be used very often, so I am ok with dropping this "nice-to-have" functionality if we would prefer that. Then our users will need to listen to `onOpen` and `onClose` to apply the active-state themselves.

Comment (TomSchrier) [reply packages/components/components/elvis-popover/src/react/elvia-popover.tsx:90]: Maybe we should drop this then? Then it is the same as Context Menu and its "button pressed"-state:
Skjermbilde 2024-04-25 kl  15 00 05
On the other hand, we will need to document `'e-text-abbreviation---active'`

File: packages/web/src/app/doc-pages/brand/typography-doc/abbreviation-example-ceg/abbreviation-example-ceg.component.scss
Comment (TomSchrier) [inline packages/web/src/app/doc-pages/brand/typography-doc/abbreviation-example-ceg/abbreviation-example-ceg.component.scss:1]: Delete this for the Sonar Cloud error 😄

File: packages/web/src/app/doc-pages/brand/typography-doc/abbreviation-example-ceg/abbreviation-example-ceg.component.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+import * as template from './abbreviation-example-ceg.component.html';
+import { StaticComponentExample } from 'src/app/shared/component-documentation/ceg';
+
+@Component({
+  selector: 'app-abbreviation-ceg',
+  standalone: true,
+  imports: [NgClass],
+  templateUrl: './abbreviation-example-ceg.component.html',
+  styleUrl: './abbreviation-example-ceg.component.scss',
```
Comment (TomSchrier) [inline packages/web/src/app/doc-pages/brand/typography-doc/abbreviation-example-ceg/abbreviation-example-ceg.component.ts:12]:
```suggestion
  
```

**Review by TomSchrier — APPROVED** [review]

**Pull request merged**